docs/ml-classification-regression.md

```diff
@@ -27,10 +27,10 @@ displayTitle: Classification and regression in spark.ml
 * This will become a table of contents (this text will be scraped).
 {:toc}
 
-In MLlib, we implement popular linear methods such as logistic
+In `spark.ml`, we implement popular linear methods such as logistic
```

Comment:
I see the purpose now. It was the old MLlib text, but a lot of it still applies. The distinction is removed.